2019-08-09 15:14:28,148 - DEBUG - tty.usbmodem0006822145041: Writing: 14 bytearray(b'')
2019-08-09 15:14:28,150 - DEBUG - tty.usbmodem0006822145041: TX: 010e
2019-08-09 15:14:28,575 - DEBUG - tty.usbmodem0006822145041: RX: 04810200fe
2019-08-09 15:14:28,575 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: DeviceStarted, data: {'operating_mode': 2, 'hw_error': 0, 'data_credit_available': 254}}
2019-08-09 15:14:28,576 - INFO - tty.usbmodem0006822145041: Device rebooted.
2019-08-09 15:14:28,868 - INFO - tty.usbmodem0006822145041.MeshConfigurer: 

2019-08-09 15:14:28,869 - DEBUG - tty.usbmodem0006822145041: Writing: 105 bytearray(b"\xc5K\x0b\x93\x8b{\xb3Y\x94\xb4\x12\xd0fk)e\xd23\xba\x07\xfb\xfbU\xd5\xed\xaa\xefN\xd8\x1a\x94\xf12\xcfg\xa9\x82\xf9p\xa3_I\xfb\n\x95VkRfX\x00\x04\xc3\x15\xa9\x02\x16\x87\xf6Y$T\x9c\r;\xb7\xd9)\x93:\xd7\x8b\x1d3\xb5\n\xe1\x99,\x903\xd2\x1c\xd3\x85p\x91\x07\x0c1x\xd0\x8d\'\xc1x")
2019-08-09 15:14:28,870 - INFO - tty.usbmodem0006822145041.MeshConfigurer: Sending next message
2019-08-09 15:14:28,870 - DEBUG - tty.usbmodem0006822145041: TX: 6169c54b0b938b7bb35994b412d0666b2965d233ba07fbfb55d5edaaef4ed81a94f132cf67a982f970a35f49fb0a95566b5266580004c315a9021687f65924549c0d3bb7d929933ad78b1d33b50ae1992c9033d21cd3857091070c3178d08d27c178
2019-08-09 15:14:28,871 - ERROR - tty.usbmodem0006822145041: Exception in pkt handler <bound method Mesh.cmd_handler of <interactive_pyaci.Mesh object at 0x1078fdd68>>
2019-08-09 15:14:28,872 - ERROR - tty.usbmodem0006822145041: traceback: Traceback (most recent call last):
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/aci/aci_uart.py", line 98, in process_command
    fun(command)
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/interactive_pyaci.py", line 379, in cmd_handler
    if cmd.RESPONSE_LUT[command._opcode]["name"] == "AddrPublicationAdd":
KeyError: 105

2019-08-09 15:14:28,880 - DEBUG - tty.usbmodem0006822145041: RX: 03845384
2019-08-09 15:14:28,880 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 83, 'status': 132, 'data': bytearray(b'')}}
2019-08-09 15:14:28,881 - ERROR - tty.usbmodem0006822145041: None: ERROR_INVALID_LENGTH
2019-08-09 15:14:28,882 - DEBUG - tty.usbmodem0006822145041: RX: 03840084
2019-08-09 15:14:28,883 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 0, 'status': 132, 'data': bytearray(b'')}}
2019-08-09 15:14:28,883 - ERROR - tty.usbmodem0006822145041: None: ERROR_INVALID_LENGTH
2019-08-09 15:14:28,883 - DEBUG - tty.usbmodem0006822145041: RX: 03840084
2019-08-09 15:14:28,885 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 0, 'status': 132, 'data': bytearray(b'')}}
2019-08-09 15:14:28,885 - ERROR - tty.usbmodem0006822145041: None: ERROR_INVALID_LENGTH
2019-08-09 15:14:28,892 - DEBUG - tty.usbmodem0006822145041: RX: 03840084
2019-08-09 15:14:28,893 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 0, 'status': 132, 'data': bytearray(b'')}}
2019-08-09 15:14:28,893 - ERROR - tty.usbmodem0006822145041: None: ERROR_INVALID_LENGTH
2019-08-09 15:14:28,893 - DEBUG - tty.usbmodem0006822145041: RX: 03840084
2019-08-09 15:14:28,894 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 0, 'status': 132, 'data': bytearray(b'')}}
2019-08-09 15:14:28,894 - ERROR - tty.usbmodem0006822145041: None: ERROR_INVALID_LENGTH
2019-08-09 15:14:28,895 - DEBUG - tty.usbmodem0006822145041: RX: 03840084
2019-08-09 15:14:28,895 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 0, 'status': 132, 'data': bytearray(b'')}}
2019-08-09 15:14:28,895 - ERROR - tty.usbmodem0006822145041: None: ERROR_INVALID_LENGTH
2019-08-09 15:14:28,895 - DEBUG - tty.usbmodem0006822145041: RX: 03840084
2019-08-09 15:14:28,896 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 0, 'status': 132, 'data': bytearray(b'')}}
2019-08-09 15:14:28,896 - ERROR - tty.usbmodem0006822145041: None: ERROR_INVALID_LENGTH
2019-08-09 15:14:28,896 - DEBUG - tty.usbmodem0006822145041: RX: 03840084
2019-08-09 15:14:28,896 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 0, 'status': 132, 'data': bytearray(b'')}}
2019-08-09 15:14:28,896 - ERROR - tty.usbmodem0006822145041: None: ERROR_INVALID_LENGTH
2019-08-09 15:14:28,897 - DEBUG - tty.usbmodem0006822145041: RX: 03840084
2019-08-09 15:14:28,897 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 0, 'status': 132, 'data': bytearray(b'')}}
2019-08-09 15:14:28,897 - ERROR - tty.usbmodem0006822145041: None: ERROR_INVALID_LENGTH
2019-08-09 15:14:28,897 - DEBUG - tty.usbmodem0006822145041: RX: 03840084
2019-08-09 15:14:28,897 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 0, 'status': 132, 'data': bytearray(b'')}}
2019-08-09 15:14:28,898 - ERROR - tty.usbmodem0006822145041: None: ERROR_INVALID_LENGTH
2019-08-09 15:14:28,898 - DEBUG - tty.usbmodem0006822145041: RX: 03840084
2019-08-09 15:14:28,898 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 0, 'status': 132, 'data': bytearray(b'')}}
2019-08-09 15:14:28,898 - ERROR - tty.usbmodem0006822145041: None: ERROR_INVALID_LENGTH
2019-08-09 15:14:28,898 - DEBUG - tty.usbmodem0006822145041: RX: 03840084
2019-08-09 15:14:28,900 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 0, 'status': 132, 'data': bytearray(b'')}}
2019-08-09 15:14:28,900 - ERROR - tty.usbmodem0006822145041: None: ERROR_INVALID_LENGTH
2019-08-09 15:14:28,901 - DEBUG - tty.usbmodem0006822145041: RX: 03840084
2019-08-09 15:14:28,901 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 0, 'status': 132, 'data': bytearray(b'')}}
2019-08-09 15:14:28,901 - ERROR - tty.usbmodem0006822145041: None: ERROR_INVALID_LENGTH
2019-08-09 15:14:28,904 - DEBUG - tty.usbmodem0006822145041: Writing: 159 bytearray(b'\x01\x00\x01\x00')
2019-08-09 15:14:28,905 - DEBUG - tty.usbmodem0006822145041: TX: 059f01000100
2019-08-09 15:14:28,905 - ERROR - tty.usbmodem0006822145041: Exception in pkt handler <bound method Mesh.cmd_handler of <interactive_pyaci.Mesh object at 0x1078fdd68>>
2019-08-09 15:14:28,906 - ERROR - tty.usbmodem0006822145041: traceback: Traceback (most recent call last):
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/aci/aci_uart.py", line 98, in process_command
    fun(command)
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/interactive_pyaci.py", line 379, in cmd_handler
    if cmd.RESPONSE_LUT[command._opcode]["name"] == "AddrPublicationAdd":
KeyError: 159

2019-08-09 15:14:28,907 - DEBUG - tty.usbmodem0006822145041: RX: 03849f00
2019-08-09 15:14:28,907 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 159, 'status': 0, 'data': bytearray(b'')}}
2019-08-09 15:14:28,908 - INFO - tty.usbmodem0006822145041: Success
2019-08-09 15:14:28,927 - DEBUG - tty.usbmodem0006822145041: Writing: 146 bytearray(b'\x00\x00\x18\xee\xd9\xc2\xa5j\xdd\x85\x04\x9f\xfc<Y\xad\x0e\x12')
2019-08-09 15:14:28,927 - DEBUG - tty.usbmodem0006822145041: TX: 1392000018eed9c2a56add85049ffc3c59ad0e12
2019-08-09 15:14:28,932 - DEBUG - tty.usbmodem0006822145041: RX: 058492000000
2019-08-09 15:14:28,932 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 146, 'status': 0, 'data': bytearray(b'\x00\x00')}}
2019-08-09 15:14:28,933 - INFO - tty.usbmodem0006822145041: SubnetAdd: {'subnet_handle': 0}
2019-08-09 15:14:28,950 - DEBUG - tty.usbmodem0006822145041: Writing: 151 bytearray(b'\x00\x00\x00\x00Oh\xad\x85\xd9\xf4\x8a\xc8X\x9d\xf6e\xb6\xb4\x9b\x8a')
2019-08-09 15:14:28,951 - DEBUG - tty.usbmodem0006822145041: TX: 1597000000004f68ad85d9f48ac8589df665b6b49b8a
2019-08-09 15:14:28,955 - DEBUG - tty.usbmodem0006822145041: RX: 058497000000
2019-08-09 15:14:28,956 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 151, 'status': 0, 'data': bytearray(b'\x00\x00')}}
2019-08-09 15:14:28,956 - INFO - tty.usbmodem0006822145041: AppkeyAdd: {'appkey_handle': 0}
2019-08-09 15:14:28,972 - DEBUG - tty.usbmodem0006822145041: Writing: 151 bytearray(b'\x01\x00\x00\x00*\xa2\xa6\xde\xd5\xa0y\x8c\xea\xb5x|\xa3\xae9\xfc')
2019-08-09 15:14:28,972 - DEBUG - tty.usbmodem0006822145041: TX: 1597010000002aa2a6ded5a0798ceab5787ca3ae39fc
2019-08-09 15:14:28,976 - DEBUG - tty.usbmodem0006822145041: RX: 058497000100
2019-08-09 15:14:28,976 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 151, 'status': 0, 'data': bytearray(b'\x01\x00')}}
2019-08-09 15:14:28,976 - INFO - tty.usbmodem0006822145041: AppkeyAdd: {'appkey_handle': 1}
2019-08-09 15:14:28,995 - DEBUG - tty.usbmodem0006822145041: Writing: 161 bytearray(b'\x01\xc0')
2019-08-09 15:14:28,995 - DEBUG - tty.usbmodem0006822145041: TX: 03a101c0
2019-08-09 15:14:28,997 - DEBUG - tty.usbmodem0006822145041: RX: 0584a1000000
2019-08-09 15:14:28,997 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 161, 'status': 0, 'data': bytearray(b'\x00\x00')}}
2019-08-09 15:14:28,998 - INFO - tty.usbmodem0006822145041: AddrSubscriptionAdd: {'address_handle': 0}
2019-08-09 15:14:29,007 - INFO - tty.usbmodem0006822145041.MeshConfigurer: 

2019-08-09 15:14:29,007 - INFO - tty.usbmodem0006822145041.MeshConfigurer: Sending next message
2019-08-09 15:14:29,016 - DEBUG - tty.usbmodem0006822145041: Writing: 164 bytearray(b'\x10\x00')
2019-08-09 15:14:29,017 - DEBUG - tty.usbmodem0006822145041: TX: 03a41000
2019-08-09 15:14:29,019 - DEBUG - tty.usbmodem0006822145041: RX: 0584a4000100
2019-08-09 15:14:29,019 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 164, 'status': 0, 'data': bytearray(b'\x01\x00')}}
2019-08-09 15:14:29,019 - INFO - tty.usbmodem0006822145041: AddrPublicationAdd: {'address_handle': 1}
2019-08-09 15:14:29,019 - ERROR - tty.usbmodem0006822145041: Exception in pkt handler <bound method Mesh.cmdRsp_handler of <interactive_pyaci.Mesh object at 0x1078fdd68>>
2019-08-09 15:14:29,020 - ERROR - tty.usbmodem0006822145041: traceback: Traceback (most recent call last):
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/aci/aci_uart.py", line 90, in process_packet
    fun(packet)
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/interactive_pyaci.py", line 280, in cmdRsp_handler
    node, element = src_address_to_node_element_index
NameError: name 'src_address_to_node_element_index' is not defined

2019-08-09 15:14:33,012 - INFO - tty.usbmodem0006822145041.MeshConfigurer: Timout handler timeout
2019-08-09 15:14:33,012 - DEBUG - tty.usbmodem0006822145041: Writing: 164 bytearray(b'\x10\x00')
2019-08-09 15:14:33,012 - DEBUG - tty.usbmodem0006822145041: TX: 03a41000
2019-08-09 15:14:33,014 - DEBUG - tty.usbmodem0006822145041: RX: 0584a4000100
2019-08-09 15:14:33,014 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 164, 'status': 0, 'data': bytearray(b'\x01\x00')}}
2019-08-09 15:14:33,015 - INFO - tty.usbmodem0006822145041: AddrPublicationAdd: {'address_handle': 1}
2019-08-09 15:14:33,015 - ERROR - tty.usbmodem0006822145041: Exception in pkt handler <bound method Mesh.cmdRsp_handler of <interactive_pyaci.Mesh object at 0x1078fdd68>>
2019-08-09 15:14:33,015 - ERROR - tty.usbmodem0006822145041: traceback: Traceback (most recent call last):
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/aci/aci_uart.py", line 90, in process_packet
    fun(packet)
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/interactive_pyaci.py", line 280, in cmdRsp_handler
    node, element = src_address_to_node_element_index
NameError: name 'src_address_to_node_element_index' is not defined

2019-08-09 15:14:37,013 - INFO - tty.usbmodem0006822145041.MeshConfigurer: Timout handler timeout
2019-08-09 15:14:37,015 - DEBUG - tty.usbmodem0006822145041: Writing: 164 bytearray(b'\x10\x00')
2019-08-09 15:14:37,015 - DEBUG - tty.usbmodem0006822145041: TX: 03a41000
2019-08-09 15:14:37,017 - DEBUG - tty.usbmodem0006822145041: RX: 0584a4000100
2019-08-09 15:14:37,018 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 164, 'status': 0, 'data': bytearray(b'\x01\x00')}}
2019-08-09 15:14:37,018 - INFO - tty.usbmodem0006822145041: AddrPublicationAdd: {'address_handle': 1}
2019-08-09 15:14:37,018 - ERROR - tty.usbmodem0006822145041: Exception in pkt handler <bound method Mesh.cmdRsp_handler of <interactive_pyaci.Mesh object at 0x1078fdd68>>
2019-08-09 15:14:37,019 - ERROR - tty.usbmodem0006822145041: traceback: Traceback (most recent call last):
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/aci/aci_uart.py", line 90, in process_packet
    fun(packet)
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/interactive_pyaci.py", line 280, in cmdRsp_handler
    node, element = src_address_to_node_element_index
NameError: name 'src_address_to_node_element_index' is not defined

2019-08-09 15:14:40,766 - DEBUG - tty.usbmodem0006822145041: Writing: 97 bytearray(b'')
2019-08-09 15:14:40,777 - DEBUG - tty.usbmodem0006822145041: TX: 0161
2019-08-09 15:14:40,778 - ERROR - tty.usbmodem0006822145041: Exception in pkt handler <bound method Mesh.cmd_handler of <interactive_pyaci.Mesh object at 0x1078fdd68>>
2019-08-09 15:14:40,780 - DEBUG - tty.usbmodem0006822145041: RX: 03846100
2019-08-09 15:14:40,780 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 97, 'status': 0, 'data': bytearray(b'')}}
2019-08-09 15:14:40,780 - INFO - tty.usbmodem0006822145041: Success
2019-08-09 15:14:40,785 - ERROR - tty.usbmodem0006822145041: traceback: Traceback (most recent call last):
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/aci/aci_uart.py", line 98, in process_command
    fun(command)
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/interactive_pyaci.py", line 379, in cmd_handler
    if cmd.RESPONSE_LUT[command._opcode]["name"] == "AddrPublicationAdd":
KeyError: 97

2019-08-09 15:14:41,019 - INFO - tty.usbmodem0006822145041.MeshConfigurer: Timout handler timeout
2019-08-09 15:14:41,029 - DEBUG - tty.usbmodem0006822145041: Writing: 164 bytearray(b'\x10\x00')
2019-08-09 15:14:41,029 - DEBUG - tty.usbmodem0006822145041: TX: 03a41000
2019-08-09 15:14:41,031 - DEBUG - tty.usbmodem0006822145041: RX: 0584a4000100
2019-08-09 15:14:41,032 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 164, 'status': 0, 'data': bytearray(b'\x01\x00')}}
2019-08-09 15:14:41,032 - INFO - tty.usbmodem0006822145041: AddrPublicationAdd: {'address_handle': 1}
2019-08-09 15:14:41,032 - ERROR - tty.usbmodem0006822145041: Exception in pkt handler <bound method Mesh.cmdRsp_handler of <interactive_pyaci.Mesh object at 0x1078fdd68>>
2019-08-09 15:14:41,033 - ERROR - tty.usbmodem0006822145041: traceback: Traceback (most recent call last):
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/aci/aci_uart.py", line 90, in process_packet
    fun(packet)
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/interactive_pyaci.py", line 280, in cmdRsp_handler
    node, element = self.src_address_to_node_element_index
NameError: name 'src_address_to_node_element_index' is not defined

2019-08-09 15:14:45,067 - INFO - tty.usbmodem0006822145041.MeshConfigurer: Timout handler timeout
2019-08-09 15:14:45,076 - DEBUG - tty.usbmodem0006822145041: Writing: 164 bytearray(b'\x10\x00')
2019-08-09 15:14:45,076 - DEBUG - tty.usbmodem0006822145041: TX: 03a41000
2019-08-09 15:14:45,078 - DEBUG - tty.usbmodem0006822145041: RX: 0584a4000100
2019-08-09 15:14:45,079 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 164, 'status': 0, 'data': bytearray(b'\x01\x00')}}
2019-08-09 15:14:45,079 - INFO - tty.usbmodem0006822145041: AddrPublicationAdd: {'address_handle': 1}
2019-08-09 15:14:45,079 - ERROR - tty.usbmodem0006822145041: Exception in pkt handler <bound method Mesh.cmdRsp_handler of <interactive_pyaci.Mesh object at 0x1078fdd68>>
2019-08-09 15:14:45,079 - ERROR - tty.usbmodem0006822145041: traceback: Traceback (most recent call last):
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/aci/aci_uart.py", line 90, in process_packet
    fun(packet)
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/interactive_pyaci.py", line 280, in cmdRsp_handler
    node, element = self.src_address_to_node_element_index
NameError: name 'src_address_to_node_element_index' is not defined

2019-08-09 15:14:49,076 - INFO - tty.usbmodem0006822145041.MeshConfigurer: Timout handler timeout
2019-08-09 15:14:49,079 - DEBUG - tty.usbmodem0006822145041: Writing: 164 bytearray(b'\x10\x00')
2019-08-09 15:14:49,080 - DEBUG - tty.usbmodem0006822145041: TX: 03a41000
2019-08-09 15:14:49,082 - DEBUG - tty.usbmodem0006822145041: RX: 0584a4000100
2019-08-09 15:14:49,082 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 164, 'status': 0, 'data': bytearray(b'\x01\x00')}}
2019-08-09 15:14:49,083 - INFO - tty.usbmodem0006822145041: AddrPublicationAdd: {'address_handle': 1}
2019-08-09 15:14:49,083 - ERROR - tty.usbmodem0006822145041: Exception in pkt handler <bound method Mesh.cmdRsp_handler of <interactive_pyaci.Mesh object at 0x1078fdd68>>
2019-08-09 15:14:49,084 - ERROR - tty.usbmodem0006822145041: traceback: Traceback (most recent call last):
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/aci/aci_uart.py", line 90, in process_packet
    fun(packet)
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/interactive_pyaci.py", line 280, in cmdRsp_handler
    node, element = self.src_address_to_node_element_index
NameError: name 'src_address_to_node_element_index' is not defined

2019-08-09 15:14:49,290 - DEBUG - tty.usbmodem0006822145041: exited read event
2019-08-09 15:15:00,285 - DEBUG - tty.usbmodem0006822145041: Writing: 14 bytearray(b'')
2019-08-09 15:15:00,291 - DEBUG - tty.usbmodem0006822145041: TX: 010e
2019-08-09 15:15:00,711 - DEBUG - tty.usbmodem0006822145041: RX: 04810200fe
2019-08-09 15:15:00,711 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: DeviceStarted, data: {'operating_mode': 2, 'hw_error': 0, 'data_credit_available': 254}}
2019-08-09 15:15:00,711 - INFO - tty.usbmodem0006822145041: Device rebooted.
2019-08-09 15:15:00,846 - INFO - tty.usbmodem0006822145041.MeshConfigurer: 

2019-08-09 15:15:00,847 - INFO - tty.usbmodem0006822145041.MeshConfigurer: Sending next message
2019-08-09 15:15:00,848 - DEBUG - tty.usbmodem0006822145041: Writing: 105 bytearray(b'\x93m\xcd\xfc:j\xf3U1@aS\x04\x96?\x8b\x8cK\xbdq\x118>\xaa\xf4\x0e\xdbrb\xed\xdbk\xf7\xc2\xe7\x84\xd6\x1f\xcaO\xce\x81G\x87\x9aA9\x1eH\xd0\xb5\xfd\x85oj\xabg0\x05\x0e\x95\xe3_\rz/\xb61RN\x11\x8e\xf3\x89\x15\x1d\xaf\x8f\xafA\xc2\xb18z\x0e\xc5\xef\xbc\xd2t\xc1Bx\x0fo#')
2019-08-09 15:15:00,849 - DEBUG - tty.usbmodem0006822145041: TX: 6169936dcdfc3a6af3553140615304963f8b8c4bbd7111383eaaf40edb7262eddb6bf7c2e784d61fca4fce8147879a41391e48d0b5fd856f6aab6730050e95e35f0d7a2fb631524e118ef389151daf8faf41c2b1387a0ec5efbcd274c142780f6f23
2019-08-09 15:15:00,852 - ERROR - tty.usbmodem0006822145041: Exception in pkt handler <bound method Mesh.cmd_handler of <interactive_pyaci.Mesh object at 0x103296c18>>
2019-08-09 15:15:00,853 - ERROR - tty.usbmodem0006822145041: traceback: Traceback (most recent call last):
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/aci/aci_uart.py", line 98, in process_command
    fun(command)
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/interactive_pyaci.py", line 379, in cmd_handler
    if cmd.RESPONSE_LUT[command._opcode]["name"] == "AddrPublicationAdd":
KeyError: 105

2019-08-09 15:15:00,861 - DEBUG - tty.usbmodem0006822145041: RX: 03845384
2019-08-09 15:15:00,861 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 83, 'status': 132, 'data': bytearray(b'')}}
2019-08-09 15:15:00,861 - ERROR - tty.usbmodem0006822145041: None: ERROR_INVALID_LENGTH
2019-08-09 15:15:00,862 - DEBUG - tty.usbmodem0006822145041: RX: 03840084
2019-08-09 15:15:00,862 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 0, 'status': 132, 'data': bytearray(b'')}}
2019-08-09 15:15:00,862 - ERROR - tty.usbmodem0006822145041: None: ERROR_INVALID_LENGTH
2019-08-09 15:15:00,863 - DEBUG - tty.usbmodem0006822145041: RX: 03840084
2019-08-09 15:15:00,863 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 0, 'status': 132, 'data': bytearray(b'')}}
2019-08-09 15:15:00,863 - ERROR - tty.usbmodem0006822145041: None: ERROR_INVALID_LENGTH
2019-08-09 15:15:00,875 - DEBUG - tty.usbmodem0006822145041: RX: 03840084
2019-08-09 15:15:00,875 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 0, 'status': 132, 'data': bytearray(b'')}}
2019-08-09 15:15:00,875 - ERROR - tty.usbmodem0006822145041: None: ERROR_INVALID_LENGTH
2019-08-09 15:15:00,876 - DEBUG - tty.usbmodem0006822145041: RX: 03840084
2019-08-09 15:15:00,876 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 0, 'status': 132, 'data': bytearray(b'')}}
2019-08-09 15:15:00,876 - ERROR - tty.usbmodem0006822145041: None: ERROR_INVALID_LENGTH
2019-08-09 15:15:00,876 - DEBUG - tty.usbmodem0006822145041: RX: 03840084
2019-08-09 15:15:00,877 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 0, 'status': 132, 'data': bytearray(b'')}}
2019-08-09 15:15:00,877 - ERROR - tty.usbmodem0006822145041: None: ERROR_INVALID_LENGTH
2019-08-09 15:15:00,878 - DEBUG - tty.usbmodem0006822145041: RX: 03840084
2019-08-09 15:15:00,878 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 0, 'status': 132, 'data': bytearray(b'')}}
2019-08-09 15:15:00,879 - ERROR - tty.usbmodem0006822145041: None: ERROR_INVALID_LENGTH
2019-08-09 15:15:00,879 - DEBUG - tty.usbmodem0006822145041: RX: 03840084
2019-08-09 15:15:00,879 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 0, 'status': 132, 'data': bytearray(b'')}}
2019-08-09 15:15:00,879 - ERROR - tty.usbmodem0006822145041: None: ERROR_INVALID_LENGTH
2019-08-09 15:15:00,880 - DEBUG - tty.usbmodem0006822145041: RX: 03840084
2019-08-09 15:15:00,880 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 0, 'status': 132, 'data': bytearray(b'')}}
2019-08-09 15:15:00,880 - ERROR - tty.usbmodem0006822145041: None: ERROR_INVALID_LENGTH
2019-08-09 15:15:00,880 - DEBUG - tty.usbmodem0006822145041: RX: 03840084
2019-08-09 15:15:00,881 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 0, 'status': 132, 'data': bytearray(b'')}}
2019-08-09 15:15:00,881 - ERROR - tty.usbmodem0006822145041: None: ERROR_INVALID_LENGTH
2019-08-09 15:15:00,885 - DEBUG - tty.usbmodem0006822145041: Writing: 159 bytearray(b'\x01\x00\x01\x00')
2019-08-09 15:15:00,893 - DEBUG - tty.usbmodem0006822145041: RX: 03840084
2019-08-09 15:15:00,894 - DEBUG - tty.usbmodem0006822145041: TX: 059f01000100
2019-08-09 15:15:00,895 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 0, 'status': 132, 'data': bytearray(b'')}}
2019-08-09 15:15:00,895 - ERROR - tty.usbmodem0006822145041: None: ERROR_INVALID_LENGTH
2019-08-09 15:15:00,895 - ERROR - tty.usbmodem0006822145041: Exception in pkt handler <bound method Mesh.cmd_handler of <interactive_pyaci.Mesh object at 0x103296c18>>
2019-08-09 15:15:00,896 - DEBUG - tty.usbmodem0006822145041: RX: 03840084
2019-08-09 15:15:00,896 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 0, 'status': 132, 'data': bytearray(b'')}}
2019-08-09 15:15:00,896 - ERROR - tty.usbmodem0006822145041: traceback: Traceback (most recent call last):
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/aci/aci_uart.py", line 98, in process_command
    fun(command)
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/interactive_pyaci.py", line 379, in cmd_handler
    if cmd.RESPONSE_LUT[command._opcode]["name"] == "AddrPublicationAdd":
KeyError: 159

2019-08-09 15:15:00,896 - ERROR - tty.usbmodem0006822145041: None: ERROR_INVALID_LENGTH
2019-08-09 15:15:00,897 - DEBUG - tty.usbmodem0006822145041: RX: 03840084
2019-08-09 15:15:00,897 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 0, 'status': 132, 'data': bytearray(b'')}}
2019-08-09 15:15:00,911 - ERROR - tty.usbmodem0006822145041: None: ERROR_INVALID_LENGTH
2019-08-09 15:15:00,911 - DEBUG - tty.usbmodem0006822145041: RX: 03849f00
2019-08-09 15:15:00,911 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 159, 'status': 0, 'data': bytearray(b'')}}
2019-08-09 15:15:00,911 - INFO - tty.usbmodem0006822145041: Success
2019-08-09 15:15:00,920 - DEBUG - tty.usbmodem0006822145041: Writing: 146 bytearray(b'\x00\x00\x18\xee\xd9\xc2\xa5j\xdd\x85\x04\x9f\xfc<Y\xad\x0e\x12')
2019-08-09 15:15:00,920 - DEBUG - tty.usbmodem0006822145041: TX: 1392000018eed9c2a56add85049ffc3c59ad0e12
2019-08-09 15:15:00,924 - DEBUG - tty.usbmodem0006822145041: RX: 058492000000
2019-08-09 15:15:00,924 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 146, 'status': 0, 'data': bytearray(b'\x00\x00')}}
2019-08-09 15:15:00,924 - INFO - tty.usbmodem0006822145041: SubnetAdd: {'subnet_handle': 0}
2019-08-09 15:15:00,944 - DEBUG - tty.usbmodem0006822145041: Writing: 151 bytearray(b'\x00\x00\x00\x00Oh\xad\x85\xd9\xf4\x8a\xc8X\x9d\xf6e\xb6\xb4\x9b\x8a')
2019-08-09 15:15:00,944 - DEBUG - tty.usbmodem0006822145041: TX: 1597000000004f68ad85d9f48ac8589df665b6b49b8a
2019-08-09 15:15:00,948 - DEBUG - tty.usbmodem0006822145041: RX: 058497000000
2019-08-09 15:15:00,948 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 151, 'status': 0, 'data': bytearray(b'\x00\x00')}}
2019-08-09 15:15:00,949 - INFO - tty.usbmodem0006822145041: AppkeyAdd: {'appkey_handle': 0}
2019-08-09 15:15:00,964 - DEBUG - tty.usbmodem0006822145041: Writing: 151 bytearray(b'\x01\x00\x00\x00*\xa2\xa6\xde\xd5\xa0y\x8c\xea\xb5x|\xa3\xae9\xfc')
2019-08-09 15:15:00,965 - DEBUG - tty.usbmodem0006822145041: TX: 1597010000002aa2a6ded5a0798ceab5787ca3ae39fc
2019-08-09 15:15:00,969 - DEBUG - tty.usbmodem0006822145041: RX: 058497000100
2019-08-09 15:15:00,969 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 151, 'status': 0, 'data': bytearray(b'\x01\x00')}}
2019-08-09 15:15:00,969 - INFO - tty.usbmodem0006822145041: AppkeyAdd: {'appkey_handle': 1}
2019-08-09 15:15:00,990 - DEBUG - tty.usbmodem0006822145041: Writing: 161 bytearray(b'\x01\xc0')
2019-08-09 15:15:00,990 - DEBUG - tty.usbmodem0006822145041: TX: 03a101c0
2019-08-09 15:15:00,992 - DEBUG - tty.usbmodem0006822145041: RX: 0584a1000000
2019-08-09 15:15:00,992 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 161, 'status': 0, 'data': bytearray(b'\x00\x00')}}
2019-08-09 15:15:00,993 - INFO - tty.usbmodem0006822145041: AddrSubscriptionAdd: {'address_handle': 0}
2019-08-09 15:15:01,002 - INFO - tty.usbmodem0006822145041.MeshConfigurer: 

2019-08-09 15:15:01,003 - INFO - tty.usbmodem0006822145041.MeshConfigurer: Sending next message
2019-08-09 15:15:01,016 - DEBUG - tty.usbmodem0006822145041: Writing: 164 bytearray(b'\x10\x00')
2019-08-09 15:15:01,016 - DEBUG - tty.usbmodem0006822145041: TX: 03a41000
2019-08-09 15:15:01,018 - DEBUG - tty.usbmodem0006822145041: RX: 0584a4000100
2019-08-09 15:15:01,018 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 164, 'status': 0, 'data': bytearray(b'\x01\x00')}}
2019-08-09 15:15:01,018 - INFO - tty.usbmodem0006822145041: AddrPublicationAdd: {'address_handle': 1}
2019-08-09 15:15:01,019 - ERROR - tty.usbmodem0006822145041: Exception in pkt handler <bound method Mesh.cmdRsp_handler of <interactive_pyaci.Mesh object at 0x103296c18>>
2019-08-09 15:15:01,020 - ERROR - tty.usbmodem0006822145041: traceback: Traceback (most recent call last):
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/aci/aci_uart.py", line 90, in process_packet
    fun(packet)
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/interactive_pyaci.py", line 280, in cmdRsp_handler
    node, element = self.src_address_to_node_element_index
TypeError: cannot unpack non-iterable method object

2019-08-09 15:15:05,006 - INFO - tty.usbmodem0006822145041.MeshConfigurer: Timout handler timeout
2019-08-09 15:15:05,008 - DEBUG - tty.usbmodem0006822145041: Writing: 164 bytearray(b'\x10\x00')
2019-08-09 15:15:05,008 - DEBUG - tty.usbmodem0006822145041: TX: 03a41000
2019-08-09 15:15:05,010 - DEBUG - tty.usbmodem0006822145041: RX: 0584a4000100
2019-08-09 15:15:05,010 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 164, 'status': 0, 'data': bytearray(b'\x01\x00')}}
2019-08-09 15:15:05,010 - INFO - tty.usbmodem0006822145041: AddrPublicationAdd: {'address_handle': 1}
2019-08-09 15:15:05,010 - ERROR - tty.usbmodem0006822145041: Exception in pkt handler <bound method Mesh.cmdRsp_handler of <interactive_pyaci.Mesh object at 0x103296c18>>
2019-08-09 15:15:05,010 - ERROR - tty.usbmodem0006822145041: traceback: Traceback (most recent call last):
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/aci/aci_uart.py", line 90, in process_packet
    fun(packet)
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/interactive_pyaci.py", line 280, in cmdRsp_handler
    node, element = self.src_address_to_node_element_index
TypeError: cannot unpack non-iterable method object

2019-08-09 15:15:09,009 - INFO - tty.usbmodem0006822145041.MeshConfigurer: Timout handler timeout
2019-08-09 15:15:09,009 - DEBUG - tty.usbmodem0006822145041: Writing: 164 bytearray(b'\x10\x00')
2019-08-09 15:15:09,009 - DEBUG - tty.usbmodem0006822145041: TX: 03a41000
2019-08-09 15:15:09,012 - DEBUG - tty.usbmodem0006822145041: RX: 0584a4000100
2019-08-09 15:15:09,012 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 164, 'status': 0, 'data': bytearray(b'\x01\x00')}}
2019-08-09 15:15:09,013 - INFO - tty.usbmodem0006822145041: AddrPublicationAdd: {'address_handle': 1}
2019-08-09 15:15:09,013 - ERROR - tty.usbmodem0006822145041: Exception in pkt handler <bound method Mesh.cmdRsp_handler of <interactive_pyaci.Mesh object at 0x103296c18>>
2019-08-09 15:15:09,013 - ERROR - tty.usbmodem0006822145041: traceback: Traceback (most recent call last):
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/aci/aci_uart.py", line 90, in process_packet
    fun(packet)
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/interactive_pyaci.py", line 280, in cmdRsp_handler
    node, element = self.src_address_to_node_element_index
TypeError: cannot unpack non-iterable method object

2019-08-09 15:15:13,014 - INFO - tty.usbmodem0006822145041.MeshConfigurer: Timout handler timeout
2019-08-09 15:15:13,015 - DEBUG - tty.usbmodem0006822145041: Writing: 164 bytearray(b'\x10\x00')
2019-08-09 15:15:13,016 - DEBUG - tty.usbmodem0006822145041: TX: 03a41000
2019-08-09 15:15:13,018 - DEBUG - tty.usbmodem0006822145041: RX: 0584a4000100
2019-08-09 15:15:13,018 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 164, 'status': 0, 'data': bytearray(b'\x01\x00')}}
2019-08-09 15:15:13,019 - INFO - tty.usbmodem0006822145041: AddrPublicationAdd: {'address_handle': 1}
2019-08-09 15:15:13,019 - ERROR - tty.usbmodem0006822145041: Exception in pkt handler <bound method Mesh.cmdRsp_handler of <interactive_pyaci.Mesh object at 0x103296c18>>
2019-08-09 15:15:13,019 - ERROR - tty.usbmodem0006822145041: traceback: Traceback (most recent call last):
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/aci/aci_uart.py", line 90, in process_packet
    fun(packet)
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/interactive_pyaci.py", line 280, in cmdRsp_handler
    node, element = self.src_address_to_node_element_index
TypeError: cannot unpack non-iterable method object

2019-08-09 15:15:17,020 - INFO - tty.usbmodem0006822145041.MeshConfigurer: Timout handler timeout
2019-08-09 15:15:17,022 - DEBUG - tty.usbmodem0006822145041: Writing: 164 bytearray(b'\x10\x00')
2019-08-09 15:15:17,023 - DEBUG - tty.usbmodem0006822145041: TX: 03a41000
2019-08-09 15:15:17,026 - DEBUG - tty.usbmodem0006822145041: RX: 0584a4000100
2019-08-09 15:15:17,026 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 164, 'status': 0, 'data': bytearray(b'\x01\x00')}}
2019-08-09 15:15:17,027 - INFO - tty.usbmodem0006822145041: AddrPublicationAdd: {'address_handle': 1}
2019-08-09 15:15:17,027 - ERROR - tty.usbmodem0006822145041: Exception in pkt handler <bound method Mesh.cmdRsp_handler of <interactive_pyaci.Mesh object at 0x103296c18>>
2019-08-09 15:15:17,027 - ERROR - tty.usbmodem0006822145041: traceback: Traceback (most recent call last):
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/aci/aci_uart.py", line 90, in process_packet
    fun(packet)
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/interactive_pyaci.py", line 280, in cmdRsp_handler
    node, element = self.src_address_to_node_element_index
TypeError: cannot unpack non-iterable method object

2019-08-09 15:15:21,028 - INFO - tty.usbmodem0006822145041.MeshConfigurer: Timout handler timeout
2019-08-09 15:15:21,028 - DEBUG - tty.usbmodem0006822145041: Writing: 164 bytearray(b'\x10\x00')
2019-08-09 15:15:21,029 - DEBUG - tty.usbmodem0006822145041: TX: 03a41000
2019-08-09 15:15:21,031 - DEBUG - tty.usbmodem0006822145041: RX: 0584a4000100
2019-08-09 15:15:21,031 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 164, 'status': 0, 'data': bytearray(b'\x01\x00')}}
2019-08-09 15:15:21,032 - INFO - tty.usbmodem0006822145041: AddrPublicationAdd: {'address_handle': 1}
2019-08-09 15:15:21,032 - ERROR - tty.usbmodem0006822145041: Exception in pkt handler <bound method Mesh.cmdRsp_handler of <interactive_pyaci.Mesh object at 0x103296c18>>
2019-08-09 15:15:21,033 - ERROR - tty.usbmodem0006822145041: traceback: Traceback (most recent call last):
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/aci/aci_uart.py", line 90, in process_packet
    fun(packet)
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/interactive_pyaci.py", line 280, in cmdRsp_handler
    node, element = self.src_address_to_node_element_index
TypeError: cannot unpack non-iterable method object

2019-08-09 15:15:25,033 - INFO - tty.usbmodem0006822145041.MeshConfigurer: Timout handler timeout
2019-08-09 15:15:25,034 - DEBUG - tty.usbmodem0006822145041: Writing: 164 bytearray(b'\x10\x00')
2019-08-09 15:15:25,034 - DEBUG - tty.usbmodem0006822145041: TX: 03a41000
2019-08-09 15:15:25,036 - DEBUG - tty.usbmodem0006822145041: RX: 0584a4000100
2019-08-09 15:15:25,037 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 164, 'status': 0, 'data': bytearray(b'\x01\x00')}}
2019-08-09 15:15:25,037 - INFO - tty.usbmodem0006822145041: AddrPublicationAdd: {'address_handle': 1}
2019-08-09 15:15:25,037 - ERROR - tty.usbmodem0006822145041: Exception in pkt handler <bound method Mesh.cmdRsp_handler of <interactive_pyaci.Mesh object at 0x103296c18>>
2019-08-09 15:15:25,039 - ERROR - tty.usbmodem0006822145041: traceback: Traceback (most recent call last):
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/aci/aci_uart.py", line 90, in process_packet
    fun(packet)
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/interactive_pyaci.py", line 280, in cmdRsp_handler
    node, element = self.src_address_to_node_element_index
TypeError: cannot unpack non-iterable method object

2019-08-09 15:15:29,035 - INFO - tty.usbmodem0006822145041.MeshConfigurer: Timout handler timeout
2019-08-09 15:15:29,037 - DEBUG - tty.usbmodem0006822145041: Writing: 164 bytearray(b'\x10\x00')
2019-08-09 15:15:29,037 - DEBUG - tty.usbmodem0006822145041: TX: 03a41000
2019-08-09 15:15:29,039 - DEBUG - tty.usbmodem0006822145041: RX: 0584a4000100
2019-08-09 15:15:29,040 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 164, 'status': 0, 'data': bytearray(b'\x01\x00')}}
2019-08-09 15:15:29,040 - INFO - tty.usbmodem0006822145041: AddrPublicationAdd: {'address_handle': 1}
2019-08-09 15:15:29,040 - ERROR - tty.usbmodem0006822145041: Exception in pkt handler <bound method Mesh.cmdRsp_handler of <interactive_pyaci.Mesh object at 0x103296c18>>
2019-08-09 15:15:29,041 - ERROR - tty.usbmodem0006822145041: traceback: Traceback (most recent call last):
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/aci/aci_uart.py", line 90, in process_packet
    fun(packet)
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/interactive_pyaci.py", line 280, in cmdRsp_handler
    node, element = self.src_address_to_node_element_index
TypeError: cannot unpack non-iterable method object

2019-08-09 15:15:33,038 - INFO - tty.usbmodem0006822145041.MeshConfigurer: Timout handler timeout
2019-08-09 15:15:33,040 - DEBUG - tty.usbmodem0006822145041: Writing: 164 bytearray(b'\x10\x00')
2019-08-09 15:15:33,041 - DEBUG - tty.usbmodem0006822145041: TX: 03a41000
2019-08-09 15:15:33,043 - DEBUG - tty.usbmodem0006822145041: RX: 0584a4000100
2019-08-09 15:15:33,043 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 164, 'status': 0, 'data': bytearray(b'\x01\x00')}}
2019-08-09 15:15:33,043 - INFO - tty.usbmodem0006822145041: AddrPublicationAdd: {'address_handle': 1}
2019-08-09 15:15:33,044 - ERROR - tty.usbmodem0006822145041: Exception in pkt handler <bound method Mesh.cmdRsp_handler of <interactive_pyaci.Mesh object at 0x103296c18>>
2019-08-09 15:15:33,045 - ERROR - tty.usbmodem0006822145041: traceback: Traceback (most recent call last):
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/aci/aci_uart.py", line 90, in process_packet
    fun(packet)
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/interactive_pyaci.py", line 280, in cmdRsp_handler
    node, element = self.src_address_to_node_element_index
TypeError: cannot unpack non-iterable method object

2019-08-09 15:15:37,045 - INFO - tty.usbmodem0006822145041.MeshConfigurer: Timout handler timeout
2019-08-09 15:15:37,047 - DEBUG - tty.usbmodem0006822145041: Writing: 164 bytearray(b'\x10\x00')
2019-08-09 15:15:37,047 - DEBUG - tty.usbmodem0006822145041: TX: 03a41000
2019-08-09 15:15:37,049 - DEBUG - tty.usbmodem0006822145041: RX: 0584a4000100
2019-08-09 15:15:37,049 - DEBUG - tty.usbmodem0006822145041: parsed_packet {event: CmdRsp, data: {'opcode': 164, 'status': 0, 'data': bytearray(b'\x01\x00')}}
2019-08-09 15:15:37,049 - INFO - tty.usbmodem0006822145041: AddrPublicationAdd: {'address_handle': 1}
2019-08-09 15:15:37,050 - ERROR - tty.usbmodem0006822145041: Exception in pkt handler <bound method Mesh.cmdRsp_handler of <interactive_pyaci.Mesh object at 0x103296c18>>
2019-08-09 15:15:37,051 - ERROR - tty.usbmodem0006822145041: traceback: Traceback (most recent call last):
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/aci/aci_uart.py", line 90, in process_packet
    fun(packet)
  File "/Users/Tim-Mac/msc/bm-control/dev/nrf5SDKforMeshv310src/scripts/interactive_pyaci/interactive_pyaci.py", line 280, in cmdRsp_handler
    node, element = self.src_address_to_node_element_index(self._temp_address)
TypeError: cannot unpack non-iterable method object

2019-08-09 15:15:39,918 - DEBUG - tty.usbmodem0006822145041: exited read event
2019-08-09 15:15:41,050 - INFO - tty.usbmodem0006822145041.MeshConfigurer: Timout handler timeout
